{"uuid": "fda86cce-efc9-497e-a44f-c9b15246baf3", "children": ["29cac805-65c3-466b-a1d9-039547103778"], "befores": [{"name": "_xunit_setup_method_fixture_TestAssistedReading", "status": "broken", "statusDetails": {"message": "AttributeError: 'super' object has no attribute 'setup_method'\n", "trace": "  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\python.py\", line 715, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\AutoTest\\FgAI\\test_001\\test_assisted_read.py\", line 23, in setup_method\n    super().setup_method(method)\n"}, "attachments": [{"name": "setup_failed_20250414_100141", "source": "f5793a6a-ed98-409c-aeed-2b8896b92f8d-attachment.png", "type": "image/png"}], "start": 1744624900943, "stop": 1744624901351}], "start": 1744624900943, "stop": 1744624901460}